scripts/create-amount-converter-instance.ts

```diff
@@ -0,0 +1,112 @@
+import { assert } from 'chai'
+import { ethers, network } from 'hardhat'
+
+import fmt from '../utils/format'
+import { confirmOrAbort } from '../utils/prompt'
+import { getDeployer, verify } from '../utils/deployment'
+import { AmountConverterFactory__factory } from '../typechain-types'
+import { AmountConverterDeployedEvent } from '../typechain-types/contracts/factories/AmountConverterFactory'
+
+const AMOUNT_CONVERTER_FACTORY = ''
+
+const CONVERSION_TARGET = ''
+const ALLOWED_TOKENS_TO_SELL: string[] = []
+const ALLOWED_TOKENS_TO_BUY: string[] = []
+
+const PRICE_FEEDS_HEARTBEAT_TIMEOUTS: bigint[] = []
+
+assert(ethers.isAddress(AMOUNT_CONVERTER_FACTORY), "AMOUNT_CONVERTER_FACTORY isn't set")
+assert(ethers.isAddress(CONVERSION_TARGET), `Invalid conversion target value`)
+assert(ALLOWED_TOKENS_TO_SELL.length > 0, 'Allowed tokens to sell is empty')
+assert(ALLOWED_TOKENS_TO_BUY.length > 0, 'Allowed tokens to buy is empty')
+assert(
+  ALLOWED_TOKENS_TO_SELL.length === PRICE_FEEDS_HEARTBEAT_TIMEOUTS.length,
+  'Allowed tokens to sell and heartbeat timeouts length mismatch'
+)
+
+async function main() {
+  // prettier-ignore
+  console.log(
+      `Preparing for ${fmt.name('AmountConverter')} deployment on "${fmt.network(network.name)}" network,`,
+      `using the ${fmt.name("AmountConverterFactory")} on address ${fmt.address(AMOUNT_CONVERTER_FACTORY)} ...\n`
+  )
+
+  const deployer = await getDeployer()
+
+  console.log(`Deployment parameters:`)
+  console.log(`  * Conversion target: ${fmt.value(CONVERSION_TARGET)}`)
+  console.log(
+    `  * Allowed tokens to sell: ${fmt.value('[' + ALLOWED_TOKENS_TO_SELL.join(', ') + ']')}`
+  )
+  console.log(
+    `  * Allowed tokens to buy: ${fmt.value('[' + ALLOWED_TOKENS_TO_BUY.join(', ') + ']')}`
+  )
+  console.log(
+    `  * Price feeds heartbeat timeouts: ${fmt.value(
```

Comment:
Would be easier to read if the values were mapped with the 'sell' items (using the table, e.g.)